### PR TITLE
Feature/001 homepage

### PR DESCRIPTION
**Pull request by JochenL05**
I merged the new "dev" branch to this branch. I corrected the things that were mentioned after the code review. Please check and let me know if everything is fine now

### CODE REVIEW

File: src/components/Description/Description.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+:root {
+  --background-color: white;
+}
+
+.all {
+  padding: 100px;
```
Comment (AleksandraZochowska) [inline src/components/Description/Description.css:6]: for responsiveness reasons, I'd recommend changing to: ` padding: 100px 20px;`

File: src/components/Description/Description.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+.primary {
+  color: black;
+  font-size: 50px;
+  background: var(--background-color);
+  width: 100%;
+  height: 60px;
```
Comment (AleksandraZochowska) [inline src/components/Description/Description.css:14]: for responsiveness reasons I'd recommend removing the `height: 60px;` rule

File: src/components/Mainview/Mainview.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
+:root {
+  --component-lenght: 250px;
+}
+
+.main {
+  height: calc(var(--component-lenght)*2);
+  width:100%;
+  background-image: url("https://t4.ftcdn.net/jpg/02/37/82/35/240_F_237823505_hxD7SRiYLVC647y1Pr8vKk8zm3jzkmRI.jpg");
+  background-repeat: no-repeat;
+  background-size: 100%;
```
Comment (AleksandraZochowska) [inline src/components/Mainview/Mainview.css:10]: For responsiveness reasons, I'd recommend changing to:
`background-size: cover;
background-position: center;`

File: src/components/Mainview/Mainview.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
+:root {
+  --component-lenght: 250px;
+}
+
+.main {
+  height: calc(var(--component-lenght)*2);
```
Comment (AleksandraZochowska) [inline src/components/Mainview/Mainview.css:6]: For responsiveness reasons, I'd recommend changing to:
`height: 100vh;`

**Review by AleksandraZochowska — CHANGES_REQUESTED** [review]
All is good, just add those 4 recommended minor CSS changes & it'll be ready to merge

**Comment by JochenL05** [comment]
I changed the highlighted parts of the code and pushed it.

**Pull request merged**